src/app.tsx

```diff
@@ -113,17 +112,10 @@ window.localStorage.setItem = (key, value) => {
   try {
     setStorageItem(key, value);
   } catch (e) {
-    notifications.warning(
-      { message: <StorageFullNotification /> },
-      {
-        toastId: 'storage-warning',
-        bodyClassName: 'cursor-auto',
-        closeButton: false,
```

Comment:
These properties still apply. The user should not be able to close this notification and it should not auto hide.

Reply:
I followed the designs, but I'll revert to this behaviour.